metrics/ci_integrations/test/common/config/ci_config_test.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:ci_integration/common/config/ci_config.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('CiConfig', () {
+    const ciProjectId = 'ciProjectId';
+    const storageProjectId = 'storageProjectId';
+
+    test(
+      "shoud throw ArgumentError trying to create an instance with null ciProjectId",
+      () {
+        expect(
+          () => CiConfig(
+            ciProjectId: null,
+            storageProjectId: storageProjectId,
+          ),
+          throwsArgumentError,
+        );
+      },
+    );
+
+    test(
+      "shoud throw Argument error trying to create an instance with null storageProjectId",
+      () {
+        expect(
+          () => CiConfig(
+            ciProjectId: ciProjectId,
+            storageProjectId: null,
+          ),
+          throwsArgumentError,
+        );
+      },
+    );
```

Comment:
This tests that we cannot create the `CiConfig` instance with no `storageProjectId` given and the next one - with no `ciProjectId`. I'm not sure we should get rid of this since it is a different test case.